charts/kubeflow-trainer/values.yaml

```diff
@@ -30,6 +30,19 @@ jobset:
   install: true
   # -- String to fully override jobset release name.
   fullnameOverride: jobset
+  controller:
+    # -- Security context of the jobset controller container.
+    securityContext:
+      allowPrivilegeEscalation: false
+      readOnlyRootFilesystem: true
+      capabilities:
+        drop:
+          - ALL
+    # -- Security context of the jobset controller pods.
+    podSecurityContext:
+      runAsNonRoot: true
+      seccompProfile:
+        type: RuntimeDefault
```

Comment:
Can we push it to JobSet upstream as we discussed before ?

Reply:
I see, in Helm charts when we have dependency on another chart (e.g. JobSet), does it inherit all of the values from the original JobSet chart ?

Reply:
I think you are correct actually. 

We should inherit the defaults so I'm not sure if this is actually needed.